components/DetailsCard/index.js

```diff
@@ -56,59 +57,6 @@ export default function DetailsCard() {
     return <p>Ups! Something went wrong...</p>;
   }
 
-  async function handleEditActivity(event) {
-    event.preventDefault();
-    setIsEditMode(true);
-    const form = event.target;
-    const formData = new FormData(form);
-
-    const activityData = Object.fromEntries(formData.entries());
-
-    const response = await fetch(endpoint, {
-      method: "PUT",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify(activityData),
-    });
-    if (response.ok) {
-      mutate(endpoint);
-      setIsEditMode(false);
-      event.target.reset();
-    }
-  }
```

Comment:
I just deleted this function completly because handleSubmit in CardForm already does the job.
It's unnessecary.

Reply:
coolio, lgtm